src/modules/map/components/infoButton/InfoButton.js

```diff
@@ -0,0 +1,68 @@
+import { html } from 'lit-html';
+import { BaElement } from '../../../BaElement';
+import css from './infoButton.css';
+import { $injector } from '../../../../injection';
+
+/**
+ * Button that opens an info popup
+ * @class 
+ *@author bakir_en  
+ */
+
+export class InfoButton extends BaElement {
+
+	constructor() {
+		super();
+		const { TranslationService } = $injector.inject('TranslationService');
+		this._translationService = TranslationService;  
+	} 
+
+	/**
+     *@override 
+     */
+	createView() {
+		const translate = (key) => this._translationService.translate(key);
+
+		const items = [
+			{ name: 'Hilfe', url: 'http://www.ldbv.bayern.de/hilfe.html' },
+			{ name: 'Kontakt', url: 'https://www.ldbv.bayern.de/service/kontakt.html' },
+			{ name: 'Impressum', url: 'https://geoportal.bayern.de/geoportalbayern/seiten/impressum.html' }
+		];
+
+		const openPopup = () => {
+
+			// set created popup visible
+			var popup = this.shadowRoot.getElementById('info-popup');
+			if (popup.getAttribute('type') === 'hide') {
+				popup.setAttribute('type', 'show');
+			}
+			else{
+				popup.setAttribute('type', 'hide');
+			} 
```

Comment:
It would be nicer, if we could call methods of Popus.js, like  `open()`.
If you want to test such calls, you have to register the ba-popup -element within the `InfoButton.test.js`

Reply:
Yeah, maybe that's better that way ! I'll do that :smiley: